server/apps/users/services.py

```diff
@@ -380,3 +426,25 @@ def remove_old_default(self):
     def get_default_role_id(self):
         role = self.find_one(req=None, is_default=True)
         return role.get('_id') if role is not None else None
+
+    def on_updated(self, updates, role):
+        self.__send_notification(updates, role)
+
+    def __send_notification(self, updates, role):
+        role_id = role['_id']
+
+        role_users = superdesk.get_resource_service('users').get_users_by_role(role_id)
```

Comment:
Actually no because get_users_by_role is in UsersService, this piece of code is in RolesService.